graph_dfs_debug/graph.py

```diff
@@ -19,36 +19,40 @@ def add_vertex(self, vertex, edges=()):
         self.vertices[vertex] = set(edges)
 
```

Comment:
What happens if the vertex is already present in self.vertices? Needs an if here to handle that edge case